bin/3scale_dev_backend_worker.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require '3scale/backend'
+require 'async'
+
+Async do
+  ThreeScale::Backend::Worker.work
+rescue
+  retry
+end
```

Comment:
For me a simpler version works:

```suggestion
ThreeScale::Backend::Worker.work
```

Reply:
Then sync/async is configured by `CONFIG_REDIS_ASYNC` env var